bancorprotocol/carbon-app
# Draw Shapes

fix #1508
fix #1509
fix #1510
fix #1511
fix #1512
fix #1514

https://github.com/user-attachments/assets/7cbbff49-84e3-4b6b-a366-8e9b8d073436


What to test: 
1. Go to trade, simulator, edit price or/and edit budget
2. Try each shape, you should be able to create, select, edit & delete each of them
3. Create: Create set the menu selected and you should click on the chart area to create the shape
4. Select: After creation the shape is selected. Click outside to unselect & click on the shape to select it again
5. Edit: Move the edges to reorganize the shape, move the shape itself to translate
6. Delete: Click on the bin to delete everything or use the keyboard on the selected shape ("Delete" or "Backspace")
7. Each shape should zoom with the rest of the candlesticks
